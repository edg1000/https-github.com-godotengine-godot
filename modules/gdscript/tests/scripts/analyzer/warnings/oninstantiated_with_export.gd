extends Node

@oninstantiated @export var node: Node

func test():
	pass
