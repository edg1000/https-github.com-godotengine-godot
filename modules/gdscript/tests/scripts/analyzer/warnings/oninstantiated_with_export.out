GDTEST_OK
>> WARNING
>> Line: 3
>> ONINSTANTIATED_WITH_EXPORT
>> "@oninstantiated" will set the default value after "@export" takes effect and will override it.
